dotcom-rendering/src/experiments/tests/sign-in-gate-time-of-day.ts

```diff
@@ -0,0 +1,23 @@
+import type { ABTest } from '@guardian/ab-core';
+
+export const signInGateTimesOfDay: ABTest = {
+	id: 'SignInGateTimesOfDay',
+	start: '2023-10-23',
+	expiry: '2023-11-23',
+	author: 'Lindsey Dew',
+	description: 'Show sign in gate more frequently to users in the morning',
+	audience: 0.5,
+	audienceOffset: 0.4,
+	successMeasure: 'Increase registration and sign ins by 5%',
+	audienceCriteria:
+		'1st article of the day, max 5 articles, lower priority than consent banner, simple articles (not gallery, live etc.), not signed in, not shown after dismiss, not on help, info sections etc. Exclude iOS 9 and guardian-live-australia. Suppresses other banners, and appears over epics',
+	dataLinkNames: 'SignInGateTimesOfDay',
+	showForSensitive: false,
+	canRun: () => true,
+	variants: [
+		{
+			id: 'morning',
```

Comment:
Are we just using the main variant test as a control? The test brief links seems to ask for a Variant/Control split. 

Also I can't see where the duration and audience size has been briefed? 50% of the whole audience on this seems like a lot?